coil-core/src/commonMain/kotlin/coil3/request/imageRequests.kt

```diff
@@ -30,6 +30,15 @@ expect val ImageRequest.crossfadeMillis: Int
 
 internal const val DEFAULT_CROSSFADE_MILLIS = 200
 
+fun ImageRequest.Builder.crossfadeBetweenImages(enable: Boolean) = apply {
+    extras[crossfadeBetweenImagesKey] = enable
+}
+
+val ImageRequest.crossfadeBetweenImages: Boolean
+    get() = getExtra(crossfadeBetweenImagesKey)
+
+private val crossfadeBetweenImagesKey = Extras.Key(default = false)
```

Comment:
We should also add:

```kotlin
val Extras.Key.Companion.crossfadeBetweenImages: Extras.Key<Boolean>
    get() = crossfadeBetweenImagesKey
```

Reply:
ba668de5a748eebea68a43312027a6b31b07d27a
please check this commit